### PR TITLE
Beaufort profile and scheme

### PR DESCRIPTION
**Pull request by amosharper**
Created Beaufort profile. Created Beaufort scheme based on One Half Dark default scheme.

### CODE REVIEW

**Comment by amosharper** [comment]
Note: seems GitHub doesn't like `\\` style comments in JSON. Happy for the comments to be removed if that's preferable.

File: beaufort.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+{
+    // PowerShell Beaufort profile
+    "hidden": false,
+    "name": "Beaufort",
+    "tabTitle": "Your Tab Title",
+    "fontFace": "Cascadia Mono",
+    "padding": "16, 16, 16, 16",
+    "colorScheme": "Beaufort",
+    "useAcrylic": true,
+    "acrylicOpacity": 0.7
```
Comment (rjcarneiro) [inline beaufort.md:19]: missing a comma

Comment (amosharper) [reply beaufort.md:19]: Whoops, that's embarrassing. I chopped the JSON about a little after moving it into the MarkDown file.

**Comment by amosharper** [comment]
@rjcarneiro the HTML file got created automatically by GitHub Desktop. I assumed that was standard GitHub functionality, though I've not used it enough to be familiar. I'll fix the missing comma and remove the comments momentarily...

**Comment by amosharper** [comment]
Have updated the pull request based on your feedback.

**Comment by rjcarneiro** [comment]
Thanks @amosharper for your amazing contribution 🙏 💪

**Comment by amosharper** [comment]
In case you were curious, the culprit for the HTML was a VS Code extension, and not GitHub Desktop:

`"markdown.extension.print.onFileSave": true`

**Pull request merged**